pkgs/development/libraries/fmt/default.nix

```diff
@@ -9,6 +9,12 @@ stdenv.mkDerivation rec {
     rev = "${version}";
     sha256 = "1cd8yq8va457iir1hlf17ksx11fx2hlb8i4jml8gj1875pizm0pk";
   };
+  patches = [
+    pkgs.fetchpatch {
```

Comment:
`pkgs.fetchpatch` is also not in scope, so you need to add it to your arguments (I'd reccomend removing the `pkgs`, and adding `fetchpatch` as an argument.)